src/lib/protocols/wzry.c

```diff
@@ -0,0 +1,57 @@
+#include "ndpi_protocol_ids.h"
+
+#define NDPI_CURRENT_PROTO NDPI_PROTOCOL_WZRY
+
+#include "ndpi_api.h"
+
+static void
+ndpi_int_wzry_add_connection(struct ndpi_detection_module_struct *ndpi_struct,
+                             struct ndpi_flow_struct             *flow /* , */
+                             /* ndpi_protocol_type_t protocol_type */) {
```

Comment:
I know that you have probably copied this prototype from some other file, but you can completely remove the third parameter